content/components/filter-input.mdx

```diff
@@ -0,0 +1,318 @@
+| `updated`         | static | `Today`, `This week`, `This month`, `Oct 19, 2007`, `Before June 4, 2018`, `After June 4, 2018` | `>YYYY-MM-DD`, `YYY-MM-DD` | [View](https://user-images.githubusercontent.com/980622/204581785-61de646f-491a-41cf-b87a-dc02afbd8847.png) |
+| `extension`       | static | `md, Markdown`, `json, JavaScript Object Notation`...                                           | `md`                       | [View](https://user-images.githubusercontent.com/980622/204581698-e7c6401d-66f4-4d3b-8c23-1e6c3012654f.png) |
+
+⚠️ This list will change a lot over the next month to accomodate different teams.
+
+### Generics
+
+The `state:` qualifier has multiple variants to improve the experience for specific items:
+
+- `state`: For both pull requests and issues (no icons)
+- `state-pr`: For only pull requests (with icons)
+- `state-issue`: For only issues (with icons)
+
+Therefore it's important to use the correct qualifier as soon as you can determine if you're filtering to pull requests or issues.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204605560-47a3e722-7459-4ba9-b25b-3a7a03a44241.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Use the `state-pr` variant when filtering pull requests.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204605557-0a7f7be6-9c11-4a6e-a790-59c8a3b31f4d.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use the `state` variant when filtering pull requests.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Highlighting
+
+If invalid values are used we don't show highlighting.
+
+While this applies to all qualifiers we do understand that for the ones that depend on our api endpoints (`users`, `repositories`, `organisations`...) we can't always accurately match every result and therefore while not ideal we can always highlight those. For example when not single signed-on to a organisation some repositories might not be returned.
+
+Also consider that users might copy paste strings containing qualifiers. In this case you need to run through every qualifier to set the highlight if needed.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205069299-569989df-e6b7-410f-bcf6-5106774887a0.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Remove highlighting on values when they are invalid.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205069297-b9e03b40-0d52-4079-966d-e6d089440ab6.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't show highlighting on invalid values.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### No matches
+
+When no matches are found we stop suggesting items through the popover and don't show that there aren't any matches.
```

Comment:
I don't quite get this. Do you mean `and don't show anything`?

Reply:
Yup updated!